src/pages/projects-page/AddProjectForm.tsx

```diff
@@ -34,23 +38,30 @@ export function AddProjectForm(props: Props) {
             </Typography>
           </Grid>
           <Grid container item spacing={2} alignItems="center" xs={12}>
-            <Grid item>
-              <DictInputItem field="projectName">
+            <DictInputItem field="projectName">
+              <Grid item>
                 <Box width={320}>
                   <StringInput
                     disabled={!canAddNewProject}
                     variant="outlined"
                     label={i18n._('Project title')}
                     fullWidth
                   />
+                  <FragmentRef lens={lens} />
                 </Box>
-              </DictInputItem>
-            </Grid>
-            <Grid item>
-              <SubmitButton disabled={!canAddNewProject} variant="outlined" color="primary" startIcon={<AddIcon />}>
-                {i18n._('Add')}
-              </SubmitButton>
-            </Grid>
+              </Grid>
+              <Grid item>
+                <ProjectAddSubmitButton
```

Comment:
define `ResetInputOnSubmit` and use it this way:

```tsx
<ResetInputOnSubmit lens={lens}>
  <SubmitButton>submit</SubmitButton>
</ResetInputOnSubmit>
```

The `ResetInputOnSubmit` component should get `onSubmit` from context and provide a new `onSubmit` which call original onSubmit and reset the input